pep-0011.txt

```diff
@@ -120,6 +120,7 @@ Target Triple                  Notes                       Buildbot
 ============================== =========================== ============================================== ========
 aarch64-pc-windows-msvc                                    https://buildbot.python.org/all/#/builders/729 Steve Dower
 powerpcle-unknown-linux-gnu    glibc, clang                https://buildbot.python.org/all/#/builders/435 Victor Stinner
+s390x-unknownlinux-gnu         glibc, gcc                  https://buildbot.python.org/all/#/builders/223 Victor Stinner
```

Comment:
```suggestion
s390x-unknown-linux-gnu        glibc, gcc                  https://buildbot.python.org/all/#/builders/223 Victor Stinner
```

Reply:
Ok, so when I copy/paste an example, I pick the only one with a typo :-D I fixed the typo in the s390x platform, but also on the ``x86_64-unknownlinux-gnu`` platform.